runtime/web_worker.rs

```diff
@@ -476,7 +476,7 @@ impl WebWorker {
       ops::signal::deno_signal::init_ops_and_esm(),
       ops::tty::deno_tty::init_ops_and_esm(),
       ops::http::deno_http_runtime::init_ops_and_esm(),
-      ops::bootstrap::deno_bootstrap::init_ops_and_esm(),
+      ops::bootstrap::deno_bootstrap::init_ops_and_esm(None),
```

Comment:
I don't like it, I wish we had an easy way to add a custom function just for snapshotting :(